pcfuncs/README.md

```diff
@@ -0,0 +1,25 @@
+# Azure Function App
```

Comment:
TODO: Update, or otherwise provide light documentation for the pcfuncs project and development